manager.go

```diff
@@ -209,7 +210,8 @@ func (m *Manager) SubsystemHealthChecks(ctx context.Context) {
 
 // CloseAll stops all subsystems and closes the cloud connection.
 func (m *Manager) CloseAll() {
-	ctx, cancelFunc := context.WithTimeout(context.Background(), time.Minute)
+	// 2 min timeout. It must be higher than viam-server shutdown timeout of 90 secs.
```

Comment:
Systemd has a stop timeout of 90s by default. Thus, when doing an actual shutdown, this time limit applies more strongly than the "longer" one for viam-server itself to avoid systemd force-killing the agent itself (and considering it a failure.) If you're raising this, you'll need to also raise the limit (including some margin) in the systemd service file (and thus, in the preinstall script as well.)

Reply:
Just realized, we already have TimeoutSec set to 120seconds, so just need to raise that to probably 240s or so